contracts/utils/test/lib_bytes.ts

```diff
@@ -668,7 +668,7 @@ describe('LibBytes', () => {
     describe('writeBytesWithLength', () => {
         it('should successfully write short, nested array of bytes when it takes up the whole array', async () => {
             const testBytesOffset = new BigNumber(0);
-            const emptyByteArray = ethUtil.bufferToHex(new Buffer(shortTestBytesAsBuffer.byteLength));
+            const emptyByteArray = ethUtil.bufferToHex(Buffer.alloc(shortTestBytesAsBuffer.byteLength));
```

Comment:
Out of scope: After doing some digging on `Buffer.alloc`, it seems like we could speed our tests up a bit by refactoring them to use [`Buffer.allocUnsafe`](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_allocunsafe_size). I think that it could be done pretty easily, but it's not high pri or even necessary.